pye3d/detector_3d.py

```diff
@@ -544,6 +544,8 @@ def _prepare_result(
             projected_pupil_circle = Ellipse(np.asarray([0.0, 0.0]), 0.0, 0.0, 0.0)
 
         result["ellipse"] = ellipse2dict(projected_pupil_circle)
+        result["location"] = result["ellipse"]["center"]  # pupil center in pixels
```

Comment:
What is the reason to repeat this information? After all, it is already in result[''ellipse"].

Reply:
Both 2d and previous 3d detectors included these keys. Also, I feel that both keys (locations and diameter) are very common keys, so including them as a quality-of-life enhancement makes sense to me. I agree that it is redundant information. Therefore, I do not have a strong position on this.

Reply:
Sounds good.